chainer/functions/connection/convolution_2d.py

```diff
@@ -593,6 +602,8 @@ def convolution_2d(x, W, b=None, stride=1, pad=0, cover_all=False, **kwargs):
             operation will be executed independently. Input channel size
             :math:`c_I` and output channel size :math:`c_O` must be exactly
             divisible by this value.
+        cudnn_fast (bool): Enable fast convolution algorithm in cudnn for
```

Comment:
Do users have to use this argument in any case?
If it's only for internal use, I think we don't have to document it.

Reply:
Ok! I thought so too but I wasn't sure of how to proceed